### ISSUE TITLE
Repeatedly duplicates entire directories

### ISSUE THREAD

**Issue by capybara-overdose** [bit-team/backintime#1595]
Spent my holidays helping my parents get backups set up - 1 week later, a 2TB drive is completely full despite only backing up a 512GB SSD??!

Turns out BackInTime just keeps repeatedly copying the same folders with unchanged contents into each snapshot folder!!! So I spent my xmas setting it all up again and two days late ITS DONE THE SAME THING

How is this ever supposed to work?! isn't the whole point that it only backs up new data?!?

(no theyre definitely not links)

**Comment by aryoda** [comment]
Could you please give us more context to the issue so that we can help you?

- Which BiT version are you using?
- Are you doing the backups as user or root?
- Which backup source and target file system is used
- How is the backup profile configured?  "Local mode" or "SSH"...?

Most probably you are affected by issue #988 (we are working on this) and the work-around is described in the [FAQ](https://github.com/bit-team/backintime/blob/dev/FAQ.md#version--120-works-very-slow--unchanged-files-are-backed-up):

> use "Expert Options" -> "Paste additional options to rsync" to add the value --no-perms --no-group --no-owner in that field.

**Comment by capybara-overdose** [comment]
- 1.4.1
- user
- ext4 local disks
- local

 i already use those options

**Comment by capybara-overdose** [comment]
that issue is from **2019** ...

**Comment by capybara-overdose** [comment]
sorry version 1.2.1. or whatevers in the ubuntu repos

**Comment by capybara-overdose** [comment]
OMG I just realised already tried this POS app, i filed a bug report back in JANUARY when it did the same thing to me

what have the devs been doing, how is this still not fixed yet lol

**Comment by aryoda** [comment]
Yes I know, this bug is "old" but there is well-known work-around and before releasing a fix we need to do do a lot of tests (and fix other bugs too) so please be patient with us (we took over the maintenance of BiT last year and do this in our spare time - any help is welcome).

**Comment by aryoda** [comment]
> i already use those options

If you are absolutely sure having added these options **could you please post an anonymized profile here** so that I can check it for possible other reasons...

Updating to 1.4.1 would also be a good thing (many issues were fixed since then) but it requires manual installation...

**Edit:** And seeing the mount options of the source and target would also be helpful

**Comment by capybara-overdose** [comment]
> > i already use those options
> 
> If you are absolutely sure having added these options **could you please post an anonymized profile here** so that I can check it for possible other reasons...
> 
> Updating to 1.4.1 would also be a good thing (many issues were fixed since then) but it requires manual installation...
> 
> **Edit:** And seeing the mount options of the source and target would also be helpful

yes i DID definitely add those options

I have no idea how to do any of that. anonymised profile of what? "mount options"? and the installation instructions are gibberish - random commands that dont even work. 

why does this always end up getting lumped back on people who arent even responsible for any of it.

**Comment by aryoda** [comment]
I really want to help you and can understand that non-working things are sinewy but I would also feel better if you could let go blaming us developers (for not spending enough of our spare time?) and focus on info that helps me to diagnose the problem.

> I have no idea how to do any of that. anonymised profile of what? "mount options"?

Please just post screen shots of your settings (tabs "general", "include", "auto-remove", "options" and "expert options") where you blur or blacken sensitive names that you don't want to expose here (path names, user names...) since the whole world can read this.

Instead of screen shots you can also attach the BiT backup profile file `~/.config/backintime/config` **after** having replaced sensitive strings that you don't want to expose here (user name, path names...).

The `mount` options of the backup source ("include" tab in BiT) and target folders ("Where to save snapshots" in the general tab of BiT) can be found in the terminal via the `mount` command.

**Comment by capybara-overdose** [comment]
config.version=6
global.use_flock=false
profile1.qt.last_path=/home/********/.var/app/org.mozilla.firefox
profile1.qt.places.SortColumn=1
profile1.qt.places.SortOrder=0
profile1.qt.settingsdialog.exclude.SortColumn=1
profile1.qt.settingsdialog.exclude.SortOrder=0
profile1.qt.settingsdialog.include.SortColumn=1
profile1.qt.settingsdialog.include.SortOrder=0
profile1.schedule.custom_time=8,12,18,23
profile1.schedule.day=1
profile1.schedule.mode=0
profile1.schedule.repeatedly.period=1
profile1.schedule.repeatedly.unit=20
profile1.schedule.time=0
profile1.schedule.weekday=7
profile1.snapshots.backup_on_restore.enabled=true
profile1.snapshots.bwlimit.enabled=false
profile1.snapshots.bwlimit.value=3000
profile1.snapshots.continue_on_errors=true
profile1.snapshots.copy_links=false
profile1.snapshots.copy_unsafe_links=false
profile1.snapshots.cron.ionice=true
profile1.snapshots.cron.nice=true
profile1.snapshots.cron.redirect_stderr=false
profile1.snapshots.cron.redirect_stdout=true
profile1.snapshots.dont_remove_named_snapshots=true
profile1.snapshots.exclude.1.value=.gvfs
profile1.snapshots.exclude.2.value=.cache/*
profile1.snapshots.exclude.3.value=.thumbnails*
profile1.snapshots.exclude.4.value=*.backup*
profile1.snapshots.exclude.5.value=*~
profile1.snapshots.exclude.6.value=.dropbox*
profile1.snapshots.exclude.7.value=/proc/*
profile1.snapshots.exclude.8.value=/sys/*
profile1.snapshots.exclude.9.value=/dev/*
profile1.snapshots.exclude.10.value=/run/*
profile1.snapshots.exclude.11.value=/etc/mtab
profile1.snapshots.exclude.12.value=/var/cache/apt/archives/*.deb
profile1.snapshots.exclude.13.value=lost+found/*
profile1.snapshots.exclude.14.value=/tmp/*
profile1.snapshots.exclude.15.value=/var/tmp/*
profile1.snapshots.exclude.16.value=/var/backups/*
profile1.snapshots.exclude.17.value=.Private
profile1.snapshots.exclude.18.value=/media/********/********/lost+found
profile1.snapshots.exclude.19.value=/home/********/.bashrc
profile1.snapshots.exclude.20.value=/home/********/.bash_history
profile1.snapshots.exclude.21.value=/home/********/.local/share/gnome-boxes
profile1.snapshots.exclude.22.value=/home/********/Downloads
profile1.snapshots.exclude.23.value=/media/********/********/Software/Misc config/keys
profile1.snapshots.exclude.24.value=/media/********/********/Restricted
profile1.snapshots.exclude.bysize.enabled=false
profile1.snapshots.exclude.bysize.value=500
profile1.snapshots.exclude.size=24
profile1.snapshots.include.1.type=0
profile1.snapshots.include.1.value=/home/********
profile1.snapshots.include.2.type=0
profile1.snapshots.include.2.value=/media/********/********
profile1.snapshots.include.size=2
profile1.snapshots.local.nocache=false
profile1.snapshots.local.password.save=false
profile1.snapshots.local.password.use_cache=true
profile1.snapshots.local_encfs.path=/media/********/********
profile1.snapshots.log_level=3
profile1.snapshots.min_free_inodes.enabled=false
profile1.snapshots.min_free_inodes.value=2
profile1.snapshots.min_free_space.enabled=false
profile1.snapshots.min_free_space.unit=20
profile1.snapshots.min_free_space.value=1
profile1.snapshots.mode=local
profile1.snapshots.no_on_battery=false
profile1.snapshots.notify.enabled=true
profile1.snapshots.path=/media/********/********
profile1.snapshots.path.host=pop-os
profile1.snapshots.path.profile=1
profile1.snapshots.path.user=********
profile1.snapshots.preserve_acl=false
profile1.snapshots.preserve_xattr=false
profile1.snapshots.remove_old_snapshots.enabled=false
profile1.snapshots.remove_old_snapshots.unit=80
profile1.snapshots.remove_old_snapshots.value=10
profile1.snapshots.rsync_options.enabled=true
profile1.snapshots.rsync_options.value=--no-perms --no-group --no-owner
profile1.snapshots.smart_remove=true
profile1.snapshots.smart_remove.keep_all=2
profile1.snapshots.smart_remove.keep_one_per_day=30
profile1.snapshots.smart_remove.keep_one_per_month=24
profile1.snapshots.smart_remove.keep_one_per_week=52
profile1.snapshots.smart_remove.run_remote_in_background=false
profile1.snapshots.ssh.check_commands=true
profile1.snapshots.ssh.check_ping=true
profile1.snapshots.ssh.cipher=default
profile1.snapshots.ssh.host=
profile1.snapshots.ssh.ionice=false
profile1.snapshots.ssh.nice=false
profile1.snapshots.ssh.nocache=false
profile1.snapshots.ssh.path=
profile1.snapshots.ssh.port=22
profile1.snapshots.ssh.prefix.enabled=false
profile1.snapshots.ssh.prefix.value=PATH=/opt/bin:/opt/sbin:\$PATH
profile1.snapshots.ssh.private_key_file=
profile1.snapshots.ssh.user=********
profile1.snapshots.take_snapshot_regardless_of_changes=false
profile1.snapshots.use_checksum=false
profile1.snapshots.user_backup.ionice=false
profiles.version=1
qt.last_path=/home/********/.var/app/org.mozilla.firefox
qt.logview.height=686
qt.logview.width=1920
qt.main_window.files_view.date_width=100
qt.main_window.files_view.name_width=440
qt.main_window.files_view.size_width=100
qt.main_window.files_view.sort.ascending=true
qt.main_window.files_view.sort.column=0
qt.main_window.height=1043
qt.main_window.main_splitter_left_w=239
qt.main_window.main_splitter_right_w=717
qt.main_window.second_splitter_left_w=233
qt.main_window.second_splitter_right_w=465
qt.main_window.width=960
qt.main_window.x=960
qt.main_window.y=260
qt.show_hidden_files=true

**Comment by capybara-overdose** [comment]
"mount: /media/********/********: /dev/mapper/******************* already mounted on /media/wallflower/***********."



???????????

**Comment by aryoda** [comment]
> "mount: /media/**/**: /dev/mapper/******************* already mounted on /media/wallflower/***********."
> 
> ???????????

I need the mount options of the HDD and SSD partitions which may look like this in the long `mount` output:

```
# SSD
/dev/nvme0n1p2 on / type ext4 (rw,relatime,errors=remount-ro)
# HDD
/dev/sda1 on /media/HDD1 type ext4 (rw,nosuid,nodev,noatime,nodiratime)
# possibly the /home folder is mounted differently (another partition)
```

**Comment by capybara-overdose** [comment]
*******@pop-os:~$ SSD
SSD: command not found


" mount options of the HDD and SSD partitions which may look like this in the long mount output:"
mate i have really no idea what youre even talking about much less how to find that

the linux  community has a really bad habit of just assuming everyone who asks questions knows how to solve the problem anyway. im not a programmer and  i have nfi what youre asking.

if youre going to tell me im somehow responsible for doing the work here then give clear steps in plain language that anyone can understand without a CS degree

**Comment by capybara-overdose** [comment]
what i have just noticed thats really weird is looking at the snapshot log for one of the big duplicate snapshots, it's very short and only details copying a   couple of history files from the browser cache basically. which makes sense - changes from the days browsing


The snapshot itself though is like 74GB and full of unique copies of photos etc. none of hich are shown as identified in the copy log (even when showing "All" entries)

**Comment by aryoda** [comment]
Thanks a lot for the config file!
I assume you have removed sensitive parts of the path by an empty string so I do see only slashes here.

> profile1.snapshots.path=/media//

The backup (snapshots) are stored in a folder on the /media mount point

> profile1.snapshots.include.1.type=0
> profile1.snapshots.include.1.value=/home/
> profile1.snapshots.include.2.type=0
> profile1.snapshots.include.2.value=/media//

The folder to be backed-up are `/home/...` and `/media/...`.

It looks like every snapshot contains all previously taken snapshots too which would explain why the HDD is full now.

Are these folder names the same in your config file?
> profile1.snapshots.include.2.value=/media//
> profile1.snapshots.path=/media//

The solution would then to remove the backup "target" folder from the "include tab" in the BiT profile...




It looks like the include folder contains also the backup **target** folder on `/media/` (= where the backups are stored).

**Comment by capybara-overdose** [comment]
no its complete stuffed the formatting when i pasted it, and i have no idea why, all the sensitive stuff was supposed to be asteriks but now its gone??

iom not backing stuff up to itself, im not that dumb

ill try re post the config

why is everything to do with linux always so broken

**Comment by aryoda** [comment]
> SSD: command not found

OK, I was assuming you know Linux since BiT is a Linux-only software.
No prob, here are the steps to show me the "mount options":

0. Precondition: The HDD must be attached (if it is an external HDD)
1. Open a terminal (eg. by pressing Ctrl + Alt + T)
2. Enter the command `mount`
3. Copy the output into a comment here

**Comment by capybara-overdose** [comment]
Config copy attempt 2



config.version=6
global.use_flock=false
profile1.qt.last_path=/home/test_user/.var/app/org.mozilla.firefox
profile1.qt.places.SortColumn=1
profile1.qt.places.SortOrder=0
profile1.qt.settingsdialog.exclude.SortColumn=1
profile1.qt.settingsdialog.exclude.SortOrder=0
profile1.qt.settingsdialog.include.SortColumn=1
profile1.qt.settingsdialog.include.SortOrder=0
profile1.schedule.custom_time=8,12,18,23
profile1.schedule.day=1
profile1.schedule.mode=0
profile1.schedule.repeatedly.period=1
profile1.schedule.repeatedly.unit=20
profile1.schedule.time=0
profile1.schedule.weekday=7
profile1.snapshots.backup_on_restore.enabled=true
profile1.snapshots.bwlimit.enabled=false
profile1.snapshots.bwlimit.value=3000
profile1.snapshots.continue_on_errors=true
profile1.snapshots.copy_links=false
profile1.snapshots.copy_unsafe_links=false
profile1.snapshots.cron.ionice=true
profile1.snapshots.cron.nice=true
profile1.snapshots.cron.redirect_stderr=false
profile1.snapshots.cron.redirect_stdout=true
profile1.snapshots.dont_remove_named_snapshots=true
profile1.snapshots.exclude.1.value=.gvfs
profile1.snapshots.exclude.2.value=.cache/*
profile1.snapshots.exclude.3.value=.thumbnails*
profile1.snapshots.exclude.4.value=*.backup*
profile1.snapshots.exclude.5.value=*~
profile1.snapshots.exclude.6.value=.dropbox*
profile1.snapshots.exclude.7.value=/proc/*
profile1.snapshots.exclude.8.value=/sys/*
profile1.snapshots.exclude.9.value=/dev/*
profile1.snapshots.exclude.10.value=/run/*
profile1.snapshots.exclude.11.value=/etc/mtab
profile1.snapshots.exclude.12.value=/var/cache/apt/archives/*.deb
profile1.snapshots.exclude.13.value=lost+found/*
profile1.snapshots.exclude.14.value=/tmp/*
profile1.snapshots.exclude.15.value=/var/tmp/*
profile1.snapshots.exclude.16.value=/var/backups/*
profile1.snapshots.exclude.17.value=.Private
profile1.snapshots.exclude.18.value=/media/test_user/the_storage_disc/lost+found
profile1.snapshots.exclude.19.value=/home/test_user/.bashrc
profile1.snapshots.exclude.20.value=/home/test_user/.bash_history
profile1.snapshots.exclude.21.value=/home/test_user/.local/share/gnome-boxes
profile1.snapshots.exclude.22.value=/home/test_user/Downloads
profile1.snapshots.exclude.23.value=/media/test_user/the_storage_disc/Software/Misc config/keys
profile1.snapshots.exclude.24.value=/media/test_user/the_storage_disc/Restricted
profile1.snapshots.exclude.bysize.enabled=false
profile1.snapshots.exclude.bysize.value=500
profile1.snapshots.exclude.size=24
profile1.snapshots.include.1.type=0
profile1.snapshots.include.1.value=/home/test_user
profile1.snapshots.include.2.type=0
profile1.snapshots.include.2.value=/media/test_user/the_storage_disc
profile1.snapshots.include.size=2
profile1.snapshots.local.nocache=false
profile1.snapshots.local.password.save=false
profile1.snapshots.local.password.use_cache=true
profile1.snapshots.local_encfs.path=/media/test_user/the_backup_disc
profile1.snapshots.log_level=3
profile1.snapshots.min_free_inodes.enabled=false
profile1.snapshots.min_free_inodes.value=2
profile1.snapshots.min_free_space.enabled=false
profile1.snapshots.min_free_space.unit=20
profile1.snapshots.min_free_space.value=1
profile1.snapshots.mode=local
profile1.snapshots.no_on_battery=false
profile1.snapshots.notify.enabled=true
profile1.snapshots.path=/media/test_user/the_backup_disc
profile1.snapshots.path.host=pop-os
profile1.snapshots.path.profile=1
profile1.snapshots.path.user=test_user
profile1.snapshots.preserve_acl=false
profile1.snapshots.preserve_xattr=false
profile1.snapshots.remove_old_snapshots.enabled=false
profile1.snapshots.remove_old_snapshots.unit=80
profile1.snapshots.remove_old_snapshots.value=10
profile1.snapshots.rsync_options.enabled=true
profile1.snapshots.rsync_options.value=--no-perms --no-group --no-owner
profile1.snapshots.smart_remove=true
profile1.snapshots.smart_remove.keep_all=2
profile1.snapshots.smart_remove.keep_one_per_day=30
profile1.snapshots.smart_remove.keep_one_per_month=24
profile1.snapshots.smart_remove.keep_one_per_week=52
profile1.snapshots.smart_remove.run_remote_in_background=false
profile1.snapshots.ssh.check_commands=true
profile1.snapshots.ssh.check_ping=true
profile1.snapshots.ssh.cipher=default
profile1.snapshots.ssh.host=
profile1.snapshots.ssh.ionice=false
profile1.snapshots.ssh.nice=false
profile1.snapshots.ssh.nocache=false
profile1.snapshots.ssh.path=
profile1.snapshots.ssh.port=22
profile1.snapshots.ssh.prefix.enabled=false
profile1.snapshots.ssh.prefix.value=PATH=/opt/bin:/opt/sbin:\$PATH
profile1.snapshots.ssh.private_key_file=
profile1.snapshots.ssh.user=test_user
profile1.snapshots.take_snapshot_regardless_of_changes=false
profile1.snapshots.use_checksum=false
profile1.snapshots.user_backup.ionice=false
profiles.version=1
qt.last_path=/home/test_user/.var/app/org.mozilla.firefox
qt.logview.height=686
qt.logview.width=1920
qt.main_window.files_view.date_width=100
qt.main_window.files_view.name_width=440
qt.main_window.files_view.size_width=100
qt.main_window.files_view.sort.ascending=true
qt.main_window.files_view.sort.column=0
qt.main_window.height=1043
qt.main_window.main_splitter_left_w=239
qt.main_window.main_splitter_right_w=717
qt.main_window.second_splitter_left_w=233
qt.main_window.second_splitter_right_w=465
qt.main_window.width=960
qt.main_window.x=960
qt.main_window.y=260
qt.show_hidden_files=true

**Comment by capybara-overdose** [comment]
the_storage_disc and the_backup_disc are actual seperate discs

**Comment by capybara-overdose** [comment]
```
ysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
udev on /dev type devtmpfs (rw,nosuid,relatime,size=8054732k,nr_inodes=2013683,mode=755,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1623816k,mode=755,inode64)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
/dev/mapper/compaq_vg-root on / type ext4 (rw,noatime,errors=remount-ro)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k,inode64)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=29,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=17320)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
ramfs on /run/credentials/systemd-sysusers.service type ramfs (ro,nosuid,nodev,noexec,relatime,mode=700)
/dev/sdc3 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro)
tmpfs on /run/qemu type tmpfs (rw,nosuid,nodev,relatime,mode=755,inode64)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
net_cls on /sys/fs/cgroup/net_cls type cgroup (rw,relatime,net_cls)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,size=1623812k,nr_inodes=405953,mode=700,uid=1000,gid=1000,inode64)
gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
portal on /run/user/1000/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
/dev/mapper/compaq_storage_vg-the_storage_disc on /media/test_user/the_storage_disc type ext4 (rw,nosuid,nodev,relatime,x-gvfs-show)
/dev/mapper/compaq_backup_vg-backup_the_storage_disc on /media/test_user/the_backup_disc type ext4 (rw,nosuid,nodev,relatime,x-gvfs-show)
/dev/mapper/compaq_storage_vg-compaq_timeshift on /run/timeshift/backup type ext4 (rw,relatime)

```

**Comment by capybara-overdose** [comment]
the config is that its supposed to back up the home directory on the SSD, and a seperate storage hdd, to the backup hdd. all seperate physical drives

**Comment by aryoda** [comment]
Could you please also show me the disk usage per mount:

1. Open terminal
2. Enter command `df -BG -T`
3. Copy output into a new comment here

**Comment by capybara-overdose** [comment]
```
Filesystem                                        Type     1G-blocks  Used Available Use% Mounted on
tmpfs                                             tmpfs           2G    1G        2G   1% /run
efivarfs                                          efivarfs        1G    1G        1G  89% /sys/firmware/efi/efivars
/dev/mapper/compaq_vg-root                        ext4          167G  100G       59G  63% /
tmpfs                                             tmpfs           8G    1G        8G   1% /dev/shm
tmpfs                                             tmpfs           1G    1G        1G   1% /run/lock
/dev/sdc3                                         vfat            1G    1G        1G  28% /boot/efi
tmpfs                                             tmpfs           8G    0G        8G   0% /run/qemu
tmpfs                                             tmpfs           2G    1G        2G   5% /run/user/1000
/dev/mapper/compaq_storage_vg-the_storage_disc       ext4          947G  605G      294G  68% /media/test_user/the_storage_disc
/dev/mapper/compaq_backup_vg-backup_the_storage_disc ext4         1735G 1645G        2G 100% /media/test_user/the_backup_disc
/dev/mapper/compaq_storage_vg-compaq_timeshift    ext4           98G   30G       64G  32% /run/timeshift/backup
```

**Comment by aryoda** [comment]
Config and disks setup look fine IMHO...

- How many snapshots do you already have created roughly (you can see this list of snapshots in the BiT GUI on the left side)?
- Do the snapshots show "(WITH ERRORS)" in the GUI?

> The snapshot itself though is like 74GB and full of unique copies of photos etc. none of which are shown as identified in the copy log (even when showing "All" entries)

How do you find out the size of a snapshot?

When a complete folder is copied into the backup snapshot folder you don't see the contained files in the snapshot log of BiT (only the folder).

I suspect that there are  errors in the snapshot that cause a permanent re-backup of some folders.
I cannot diagnose this without the snapshot logs (which contain personal data like folder and file names of course).

It's 3.30 AM now here so I go to bed but I will come back with more ideas to diagnose this tomorrow.

It would be really great if you manage to install the most recent release of BiT since the error recognition is improved in the latest version. But I understand that the instructions are require some profound Linux background not to be expected for normal users (esp. git clone and more):

https://github.com/bit-team/backintime/releases

**Comment by buhtz** [comment]
Dear @capybara-overdose ,
thanks for this report.

Please take one step back and breath three times. You started very rude even with the title of your report. You are still being rude (using lower case letters only but screaming with upper case letters). Despite of that aryoda still try to help you and keeps communicating with you. I know your account from issues in the past and it seems you have not learned much about how to do a dialog in an open source project with its maintainers and supporters. Be happy that aryoda answered first. I would have blocked you from the very first beginning because I know your account.

The current maintenance team was in a similar situation like you except that we where a bit more polite. :laughing: Then we (friendly) took over the project from the former maintainer to fix the things our self.

Please also use the infrastructure of our project. You mentioned an issue you opened in the past? In that case you have to link to it. #1396 We can help you only if you provide us information. It seems you have not read the README.md file? There in the beginning the maintenance team is described. Known problems are described.

About installing a fresher version: You could use one of the [alternative sources](https://github.com/bit-team/backintime#alternative-installation-options) e.g. a PPA. Be aware that we do not support that sources. Please ask the maintainers of that PPA for assistance.

We do treat every report as an opportunity to learn about us, about the project and to improve BIT.

**Comment by aryoda** [comment]
@capybara-overdose 

To get a quick overview about the used disk space per snapshot could you please

1. open a terminal
2. navigate to the "root" folder of your snapshots (should be the command
   `cd /media/test_user/the_backup_disc/backintime/pop-os/test-user/1`
    with `test-user` as your user name (you have masked it here).
    If the command doesn't work just use
   `cd /media/test_user/the_backup_disc`
   and then repeatedly
   `cd <press tab to autocomplete>`
   until you are in the folder for the backup configuration named `1`
3. enter the command `du -h --max-depth=1` and post the output here
4. enter the command `du /home -h --max-depth=1` and post the output here
5. enter the command `du /media/test_user/the_storage_disc -h --max-depth=0` and post the output here

I can see then the used space of the snapshots vs. the used space of the backup source folders
and can pick one example that could help me to pick two snapshots to examine in detail (esp. hardlinks).

**Comment by buhtz** [comment]
I vote to close as there is no feedback and in my opinion there is no reproducible evidence of a bug.

**Comment by capybara-overdose** [comment]
"No reproducible evidence of a bug"

Is that a joke? Do you even USE this trainwreck of an app?

I just reinstalled it purely to check if this was true or you just shucking responsibility again, and it  did it on literally the first run.

You close it, someone else is just going to open another. Stop making excuses and fix it. And don't give us this "iTs OpEn SoUrCe wErE vOlUnTeErS"  cope - you chose to take this responsibility, so take it seriously. If you can't, then have the stones to come clean and tell people you've given up. Otherwise? Do your job.

**Comment by buhtz** [comment]
I am not going deeper into that discussion. I blocked that user because to protect my own health.

I vote to close the Issue because new information won't come out of this discussion.
If there really is bug another user will open an Issue and hopefully provide clear information in an socially acceptable way.

**EDIT**: Because of ongoing hazardous behavior here and in other projects I reported that user to Microsoft GitHub. But recommended not to block him but warn him. I hope he can turn his energy into something positive.
